index.js

```diff
@@ -1,18 +1,49 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
-
+//I had to add a defer to the script line in index.html so there would be HTML on the page for this query to hit, and then add h1 in front of "header" in the query so it would find the element with an id of header in an h1 tag.
 
 /***** Deliverable 2 *****/
-
+header.style.color = "aqua"
+//I used console.dir(header) in the dev console on Chrome to check for the applicable key value pairs, found the above style and color keys, and then set the color to aqua.
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const travelerDiv = document.querySelector("div.traveler")
+console.log(travelerDiv)
+travelerDiv.querySelector("h2").innerText = traveler.name
+travelerDiv.querySelector("em").innerText = traveler.nickname
+travelerDiv.querySelector("img").src = traveler.photo
```

Comment:
Nice job! Since you are not referencing those particular tags again, there is no need to set them to variables